tox.ini

```diff
@@ -1,10 +1,12 @@
 [tox]
-envlist = {py35,py38}-django{22}
+envlist = py38-django{22,30,31}
 
 [testenv]
 deps =
-    django22: -r requirements/django.txt
-    -r{toxinidir}/requirements/test.txt
+    django22: Django>=2.2,<2.3
```

Comment:
Here also I thought we'd want to keep `django.txt` to be explicit about which 2.2.x version we're testing.

Reply:
reverted the `django.txt` change.